sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/AutomaticRepairsPolicy.cs

```diff
@@ -38,12 +38,16 @@ public AutomaticRepairsPolicy()
         /// starts after the state change has completed. This helps avoid
         /// premature or accidental repairs. The time duration should be
         /// specified in ISO 8601 format. The minimum allowed grace period is
-        /// 30 minutes (PT30M), which is also the default value. The maximum
+        /// 10 minutes (PT10M), which is also the default value. The maximum
         /// allowed grace period is 90 minutes (PT90M).</param>
-        public AutomaticRepairsPolicy(bool? enabled = default(bool?), string gracePeriod = default(string))
+        /// <param name="repairAction">Type of repair action (replace, restart,
+        /// reimage) that will be used for repairing unhealthy virtual machines
+        /// in the scale set. Default value is replace.</param>
+        public AutomaticRepairsPolicy(bool? enabled = default(bool?), string gracePeriod = default(string), string repairAction = default(string))
```

Comment:
>default(string) [](http://example.com/codeflow?start=130&length=15)

this defaults to null. should it be "replace" instead?

Reply:
this is generated. If needs to be updated, update need to be made in Swager